msteams-platform/m365-apps/prerequisites.md

```diff
@@ -88,24 +88,24 @@ To install Office 365 Beta Channel applications in your test environment:
 
 ### Mobile
 
-You can preview Teams personal tabs running in Office app for Android by joining the beta program.
+You can preview Teams personal tabs running in Microsoft 365 app for Android by joining the beta program.
 
-To install the latest Office app beta, build to your physical Android device or Android emulator:
+To install the latest Microsoft 365 app beta, build to your physical Android device or Android emulator:
 
 1. Ensure you use a Google Play [supported Android device](https://support.google.com/googleplay/answer/1727131).
 1. Launch the **Play Store** on your Android device.
-1. Search for office and select **Microsoft Office: Edit & Share**.
+1. Search for Microsoft 365 and select **Microsoft 365: Edit & Share**.
```

Comment:
I checked the latest, and the app is currently listed as "Office (Microsoft 365)". So let's amend to:

1. Search for "Office" and select **Office (Microsoft 365)**